apps/src/storage/GetColumnParamPicker.jsx

```diff
@@ -24,11 +25,13 @@ export default class GetColumnParamPicker extends React.Component {
   componentDidMount() {
     if (this.props.param === ParamType.COLUMN) {
       const reduxState = getStore().getState();
+      // TODO: post-firebase-cleanup, remove this conditional: #56994
+      // Only firebase needs tableType, datablock storage checks on the backend
+      const tableType = isFirebaseStorage()
+        ? reduxState.data.tableListMap[this.props.table]
+        : undefined;
       Applab.storage
-        .getColumnsForTable(
-          this.props.table,
-          reduxState.data.tableListMap[this.props.table]
-        )
+        .getColumnsForTable(this.props.table, tableType)
```

Comment:
In firebaseStorage, the frontend tracks tableType from the backend: is this a current table or a regular static table, and passes the tableType back in to all calls. This is not necessary in datablockStorage, which does the check on the backend. As a result, datablockStorage ignores tableType.